pages/index.js

```diff
@@ -18,54 +18,51 @@ let lineWidth7 = lineWidthRandomizer();
 let lineWidth8 = lineWidthRandomizer();
 
 export default function HomePage({ listOfArtworks, handleListOfArtworks }) {
-  const [canvasParameter, setCanvasParameter] = useState([
-    { color: 148 },
-    { rotation: 45 },
-    { outline: 0 },
-    {
-      lineWidthes: [
-        lineWidth1,
-        lineWidth2,
-        lineWidth3,
-        lineWidth4,
-        lineWidth5,
-        lineWidth6,
-        lineWidth7,
-        lineWidth8,
-      ],
-    },
-    { id: listOfArtworks.length + 1 },
-  ]);
+  const [canvasParameter, setCanvasParameter] = useState({
+    color: 148,
+    rotation: 45,
+    outline: 0,
+
+    lineWidthes: [
```

Comment:
It's actually spelled "widths" without the "e" in between if you care to be grammatically correct.